<!--AUTHOR: Neset Unver Akmandor (NUA)-->
<!--E-MAIL: akmandor.n@northeastern.edu-->

<launch>
  <!-- CONFIGURATION OPTIONS
    config_map_utility_jackal_ur5
  -->
  <arg name="config_map_utility"  default="config_map_utility_jackal_ur5" />

  <!-- PUBLISH CONFIGURATION PARAMETERS -->
  <rosparam file="$(find mobiman_simulation)/config/$(arg config_map_utility).yaml"/>

  <node name="map_utility_server" pkg="mobiman_simulation" type="map_utility_server" output="screen" />
</launch>